kafka-ui-react-app/src/components/Connect/List/List.tsx

```diff
@@ -43,6 +43,10 @@ const List: React.FC<ListProps> = ({
     fetchConnectors(clusterName);
   }, [fetchConnects, fetchConnectors, clusterName]);
 
+  const failedConnectors = connectors.filter((connector: FullConnectorInfo) => {
+    return connector.status.state === 'FAILED';
+  });
```

Comment:
I would suggest to move this logic to redux. Just create a new selector

Reply:
Getting failed connectors moved to redux selector and .spec file updated